packages/contracts/test/diamond/facets/StakingFacet.t.sol

```diff
@@ -333,22 +333,17 @@ contract DepositStateTest is DepositStateStaking {
         vm.prank(fourthAccount);
         stakingFacet.removeLiquidity(amount, fourthID);
 
-        assertEq(preBal + userReward, governanceToken.balanceOf(fourthAccount));
+        assertEq(preBal + userReward, 9999999999999927918000);
```

Comment:
What is `9999999999999927918000` ?

Reply:
Somehow the test started to fail after moving to latest Curve's metapool so I simply hardcoded the expected value. I haven't investigated in depth what's wrong with the staking since we have a [separate issue](https://github.com/ubiquity/ubiquity-dollar/issues/752) for that.

Reply:
perhaps good to add a comment to then not forge things later